docs/deployments/application-components.md

```diff
@@ -255,20 +256,25 @@ docker push <private-registry>/<image-namespace>/order-query-ms:latest
 ```
 
 * Generate application YAMLs via `helm template` with the following parameters:
-    - `--set image.repository=<private-registry>/<image-namespace>/<image-repository>`
-    - `--set image.tag=latest`
-    - `--set image.pullSecret=<private-registry-pullsecret>` (optional or set to blank)
-    - `--set image.pullPolicy=Always`
-    - `--set eventstreams.env=ICP`
-    - `--set eventstreams.brokersConfigMap=<kafka brokers ConfigMap name>`
-    - `--set serviceAccountName=<service-account-name>`
-    - `--namespace <target-namespace>`
-    - `--output-dir <local-template-directory>`
-
-```shell
-# Example parameters
-helm template --set image.repository=rhos-quay.internal-network.local/browncompute/order-query-ms --set image.tag=latest --set image.pullSecret= --set image.pullPolicy=Always --set eventstreams.env=ICP --set eventstreams.brokersConfigMap=kafka-brokers --set serviceAccountName=kcontainer-runtime --output-dir templ --namespace eda-refarch chart/orderqueryms/
-```
+    * `--set image.repository=<private-registry>/<image-namespace>/<image-repository>`
+    * `--set image.pullSecret=<private-registry-pullsecret>` (optional or set to blank)
+    * `--set kafka.brokersConfigMap=<kafka brokers ConfigMap name>`
+    * `--set eventstreams.enabled=(true/false)` (`true` when connecting to Event Streams of any kind, `false` when connecting to Kafka directly)
+    * `--set eventstreams.apikeyConfigMap=<kafka api key Secret name>`
+    * `--set eventstreams.truststoreRequired=(true/false)` (`true` when connecting to Event Streams via ICP4I)
+    * `--set eventstreams.truststoreSecret=<eventstreams jks file secret name>` (only used when connecting to Event Streams via ICP4I)
+    * `--set eventstreams.truststorePassword=<eventstreams jks password>` (only used when connecting to Event Streams via ICP4I)
+    * `--set serviceAccountName=<service-account-name>`
+    * `--namespace <target-namespace>`
+    * `--output-dir <local-template-directory>`
+  Example using Event Streams via ICP4I:
+   ```shell
+   helm template --set image.repository=rhos-quay.internal-network.local/browncompute/order-query-ms --set image.pullSecret= --set kafka.brokersConfigMap=es-kafka-brokers --set eventstreams.enabled=true --set eventstreams.apikeyConfigMap=es-eventstreams-apikey --set serviceAccountName=kcontainer-runtime --set eventstreams.truststoreRequired=true --set eventstreams.truststoreSecret=es-ca-pemfile --set eventstreams.truststorePassword=password --output-dir templates --namespace eda-refarch chart/orderqueryms
```

Comment:
shouldnt `eventstreams.truststoreSecret=es-ca-pemfile` be `eventstreams.truststoreSecret=es-truststore-jks`